actions/insert.py

```diff
@@ -3,6 +3,13 @@
 
 class InsertAction(BaseAction):
     def run(self, table, payload):
+        # If using new ServiceNew API...
+        if self.config['use_new_service_now_api']:
+            client = self.client
+            resource = client.resource(api_path="/table/{0}".format(table))
```

Comment:
Might be nicer to use python fstrings instead of format strings now that we are python 3 only, e.g. api_path=f"/table/{table}" for simplicity, but not essential. Applies to other places as well.

Reply:
Thanks, Amanda! This is a good suggestion; I'll keep this in mind in the future. I'm a little set in my ways about `format()` and adjusting to fstrings. :)